include/proxy-wasm/exports.h

```diff
@@ -146,6 +148,12 @@ Word wasi_unstable_args_sizes_get(Word argc_ptr, Word argv_buf_size_ptr);
 void wasi_unstable_proc_exit(Word);
 Word wasi_unstable_clock_time_get(Word, uint64_t, Word);
 Word wasi_unstable_random_get(Word, Word);
+Word wasi_unstable_fd_filestat_get(Word fd, Word statOut);
```

Comment:
It could be useful to include a comment noting that these hostcalls are included since Golang depends on their presence, otherwise it's unclear how we chose this particular subset of wasip1 to export. Though OTOH, there would be nothing preventing this documentation from going stale, and we don't (yet) document how the existing set of wasi_unstable* hostcalls was chosen. WDYT?

Reply:
I tried moving them to a separate section and commenting, but it then feels really weird to have a mixture of wasi and non-wasi hostcalls undocumented, then a few wasi hostcalls specifically documented.

I decided against commenting, as I feel the final state should be exporting all of wasip1, even if unimplemented. Does that sound acceptable?

Reply:
SGTM